Src/WitsmlExplorer.Frontend/components/ContentViews/TrajectoryView.tsx

```diff
@@ -80,7 +81,13 @@ export const TrajectoryView = (): React.ReactElement => {
     { property: "md", label: "md", type: ContentType.Number },
     { property: "incl", label: "incl", type: ContentType.Number },
     { property: "azi", label: "azi", type: ContentType.Number },
-    { property: "tvd", label: "tvd", type: ContentType.Number }
+    { property: "tvd", label: "tvd", type: ContentType.Number },
```

Comment:
Move tvd to after md to follow the [order](https://github.com/equinor/witsml-explorer/issues/183#issuecomment-1686114631).

Reply:
Fixed.